baseapp/block_gas_test.go

```diff
@@ -109,8 +110,19 @@ func TestBaseApp_BlockGas(t *testing.T) {
 				&testdata.TestMsg{},
 			)
 			genState := GenesisStateWithSingleValidator(t, cdc, appBuilder)
-			stateBytes, err := tmjson.MarshalIndent(genState, "", " ")
+
+			genStateJson := map[string]json.RawMessage{}
```

Comment:
Does it make sense to have a helper for that? This logic is repeated very often.

Reply:
Yeah, I agree. The only difference among these is the indentation of the output. Would it make sense to have something like:
`MarshalGenesisStateToJSON(gs map[string]proto.Message, cdc codec.Codec, indent bool) []byte`.
Also, where would you place this helper?

Reply:
Yeah good question 😅

Reply:
Yeah, we can add such a function @facundomedica 👍